### DISCUSSION TITLE
Could please add Studio Display?

### DISCUSSION THREAD

**Discussion by scxiayi** [emoacht/Monitorian#383] [Compatibility]
Hi, Studio Display is still unsupportable. Is there any possible to add this ? Thanks

**Comment by emoacht** [comment]
Is that display is compatible with DDC/CI? Could you post [probe.log](https://github.com/emoacht/Monitorian#probe)?

**Comment by emoacht** [comment]
I had a chance to test a Studio Display and confirmed that it is not DDC/CI compatible.
